entities/features/html_features/features_html.yml

```diff
@@ -35,6 +35,7 @@
       title: Chrome Platform Status
   tags:
     - features
+    - web-features:details-name
```

Comment:
I think maybe it should be its own separate field? It's a bit weird to use the tags field for that?